### DISCUSSION TITLE
Exporting error "Pandoc exited with code 97  Could not find data file templates/Brief.tex" with custom template

### DISCUSSION THREAD

**Discussion by Taxicletter** [Zettlr/Zettlr#5371] [Q&A] [answered]
Hi,

I made a custom template and that worked. I was on Fedora KDE. I switched to Bluefin (Fedora Silverblue) and now I get above error. I used Zettlr from Flathub.
All default templates work. One thing that may be part of the problem is that "tex" files are not recognised as files you can open with Zettlr. They're also not seen when choosing "open". I did manage to open a tex file, but I don't remember how.

Therefore I tried the Zettlr AppImage. Then Fedora can open tex files with Zettlr. But exporting results in "Pandoc exited with code 47: xelatex not found. Please select a different --pdf-engine or install xelatex" Exporting to all non-pdf formats works.

I'm no programmer, so I don't know what I can do next to fix this?

**Comment by Taxicletter** [comment]
I'd like to add: when doing this
```
flatpak install org.freedesktop.Sdk.Extension.texlive
```
I get the message texlive is already installed. So that's not the problem?

**Comment by nathanlesage** [comment, marked as answer]
The error stands for `PandocCouldNotFindDataFileError`, meaning that it couldn't find a data file required for the export. This could be an image or something within the file you try to export (or within the template – is there some data file referenced that you may have forgotten to copy over?). Otherwise, there is probably an issue with the file permissions of the Flathub Zettlr binary, [as indicated in the docs](https://docs.zettlr.com/en/getting-started/setup/#flatpak).

**Comment by Taxicletter** [comment]
Hi Nathan,
There was a picture in the tex-file, but even if I remove that, I get the same error.
Thanks for the link to the docs, I wasn't aware of that information. I installed [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal) and will try changing permissions. If the problem persists, I will ask in https://github.com/flathub/com.zettlr.Zettlr/issues.

Thank you!
EDIT:
With Flatseal I had to give Zettlr permission to access my home-folder and that fixed the problem!
